lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1302,12 +1302,8 @@ abstract class DefaultTextEditingStrategy implements TextEditingStrategy {
   }
 
   void maybeSendAction(html.Event event) {
-    if (event is html.KeyboardEvent) {
-      if (inputConfiguration.inputType.submitActionOnEnter &&
-          event.keyCode == _kReturnKeyCode) {
-        event.preventDefault();
```

Comment:
We're not preventingDefault anymore, is this so the cursor jumps to the next line in a multiline input?

PS: Wouldn't this cause a single line input to "submit the form" when the user hits enter? Do you need to still prevent default for single line inputs vs not preventing it for multiline?

Reply:
> We're not preventingDefault anymore, is this so the cursor jumps to the next line in a multiline input?

Yes. That's how the flutter behaves in other platforms.

> Wouldn't this cause a single line input to "submit the form" when the user hits enter?

Well, in our case, there's no form. It's just a single field. If the app uses autofill, then we do create a form and we preventDefault on its submit event:
https://github.com/flutter/engine/blob/c90e46300488a1335e455b999962e960c9fb1540/lib/web_ui/lib/src/engine/text_editing/text_editing.dart#L191-L193